### PR TITLE
test: Update README instructions and vm-prep

### PR DESCRIPTION
**Pull request by stefwalter**
The vm-prep command should start up libvirtd if needed.
Also add some missing dependencies to test/README

### CODE REVIEW

File: test/README [hunk 1]
```diff
@@ -5,22 +5,22 @@ files for them.
 
 To run the tests on Fedora, you need to install the following packages:
 
-    $ sudo yum install npm qemu-kvm python \
-         firewalld libvirt-client libvirt-python \
+    $ sudo yum install npm qemu-kvm qemu-system-x86 python \
```
Comment (martinpitt) [inline test/README]: qemu-kvm already pulls in -system-x86, so that looks redundant.

File: test/vm-prep [hunk 2]
```diff
@@ -42,6 +42,8 @@ silent()
 
 prepare()
 {
+    systemctl start libvirtd
```
Comment (martinpitt) [inline test/vm-prep]: This should be indented once more to line up with the code below.

**Comment by martinpitt** [comment]
Adding bot label as this only changes stuff not covered by CI (README, vm-prep).

File: test/README (hunk 1 above)
Comment (stefwalter) [inline test/README]: Thanks. Fixed.

File: test/vm-prep (hunk 2 above)
Comment (stefwalter) [inline test/vm-prep]: Fixed.

**Pull request merged**